pytket/docs/index.rst

```diff
@@ -75,18 +75,19 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
 .. _Quantinuum: https://www.quantinuum.com/
 
 .. toctree::
-    :caption: Introduction:
+    :caption: Overview:
     :maxdepth: 1
 
+    API docs home <self> 
     getting_started.rst
     changelog.rst
     install.rst
+    TKET website <https://tket.quantinuum.com/>
```

Comment:
Why are we not adding extension, examples and manual to this overview?

Reply:
Do you think its more apporiate to merge the "Overview" and "More documentation" lists?

Reply:
Yes, but if you have good reasons to not do that, I am happy to discuss that. But I am not sure what our general plan is how and if we want to add the general navigation bar on top?

Reply:
I was thinking we add these links here as an easy near term solution and then figure out the navbar later on.